Resize image transformer cannot handle images with unknown pixel format.

codemzs opened the pull request:
fixes #3611

codemzs commented:
Image with invalid format:
![taco_invalidpixelformat](https://user-images.githubusercontent.com/1211949/56843746-3fe79e00-685a-11e9-9377-d0dcc1d18c96.jpg)

Image after converting it to valid format and resizing:
![z](https://user-images.githubusercontent.com/1211949/56843750-48d86f80-685a-11e9-8bbd-d81b1249a4c4.jpg)

shauheen commented:
Similar to this [markdown](https://github.com/dotnet/machinelearning/blob/master/test/data/README.md), I think we should add a file in the images folder with info about the files. Can you please add something that shows where this image was obtained from and what the license is.

codemzs commented:
@shauheen I have updated the license information for the new taco image that I added. It seems the readme file you pointed as an images section at the bottom.

eerhardt commented on src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs:
```diff
@@ -346,11 +347,15 @@ private ValueGetter<VBuffer<TValue>> GetGetterCore<TValue>(DataViewRow input, in
+                        if (src.PixelFormat != PixelFormat.Format32bppArgb && src.PixelFormat != PixelFormat.Format24bppRgb)
+                        {
+                            var clone = src.Clone(new Rectangle(0, 0, src.Width, src.Height), PixelFormat.Format32bppArgb);
+                            src.Dispose();
```
If these need to be disposed, who disposes this clone?

codemzs replied:
@eerhardt https://github.com/dotnet/machinelearning/blob/855a7faeb84230b01a67a72e71b4d95f8d3e3bf7/src/Microsoft.ML.ImageAnalytics/ImagePixelExtractor.cs#L328
